releasenotes/notes/add-sparse-torch-connector-a3b9e3d50b405a01.yaml

```diff
@@ -0,0 +1,46 @@
+---
+features:
+  - |
+    The PyTorch connector :class:`~qiskit_machine_learning.connector.TorchConnector` now fully
+    supports sparse output in both forward and backward passes. To enable sparse support, the
+    `sparse` parameter must be set in the connector itself along with the corresponding `sparse`
+    parameter of :class:`~qiskit_machine_learning.neural_networks.SamplerQNN`. This snippet
```

Comment:
> must be set in

Since it picks up the value of the neural network anyway by default its not needed if you set in on the QNN right? Do these always have to match? In which case could we consider deprecating and dropping it there or is there some valid mix/match where conversion is done?

> corresponding `sparse`  parameter of :class:`~qiskit_machine_learning.neural_networks.SamplerQNN`.

Should this be a more general statement about a QNN class rather than SamplerQNN which is more an example - though at present I guess its really the only one we have, since EstimatorQNN does not allow sparse to be set.

Reply:
I tried to be more precise in the reno. The connector can be dense and the network is sparse, this is the only case when they don't match each other. 

No, there's no need to deprecate and drop.

And yes, there's only `SamplerQNN` that supports sparsity.